reference-architecture/aws-ansible/playbooks/roles/cloudformation-infra/files/add-infra-node.json

```diff
@@ -0,0 +1,131 @@
+{
```

Comment:
@cooktheryan I just think about how to manage the environment ongoing...  when scaling/tearing down the environment how do you know what stacks to destroy? When ASGs are in use, it is just a matter of scaling up/down the ASG through the existing stack. For multiple stacks, management is going to be wonky.